README.md

```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
 To build a single database binding:
 
     mvn -pl site.ycsb:mongodb-binding -am clean package
+
+Running multiple instances and latency percentiles
+--------------------------------------------------
+
+In general, you shall be interested in 99% percentile (P99) of the latency
+distribution and the rest - 99.9%, 99.99%, 99.999% - the latency tail.
+95% percentile has a very low probability to be observed by the user
+so don't even bother (see "How Many Nines?" at
+https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/
+). The formula to calculate probability of how many clients will
+observe a specific percentile is: 
+
+    Probability_to_observe = 1 - Percentile ^ Requests
```

Comment:
Would this be clearer as:

> Probability_to_observe = 1 - Percentile<sup>Requests</sup>

Reply:
I don't know how to do that and I did find a way to do that.